src/mc-tokens-private.h

```diff
@@ -44,6 +44,14 @@
  * ESCDerivedFromDataTokenAndCounter = HMAC(ESCDerivedFromDataToken, u)
  * ECCDerivedFromDataTokenAndCounter = HMAC(ECCDerivedFromDataToken, u)
  *
+ * EDCTwiceDerivedToken      = HMAC(EDCDerivedFromDataTokenAndCounter, 1)
+
+ * ESCTwiceDerivedTagToken   = HMAC(ESCDerivedFromDataTokenAndCounter, 1)
+ * ESCTwiceDerivedValueToken = HMAC(ESCDerivedFromDataTokenAndCounter, 2)
+
+ * ECCTwiceDerivedTagToken   = HMAC(ECCDerivedFromDataTokenAndCounter, 1)
+ * ECCTwiceDerivedValueToken = HMAC(ECCDerivedFromDataTokenAndCounter, 2)
```

Comment:
Suggest removing the ECC tokens.

I expect the ECC tokens are not needed for the V2 protocol. SERVER-74066 notes:
> In V2, since ECC is removed

ECC tokens were removed from protocol documentation in: https://github.com/10gen/mongo-enterprise-modules/commit/abe05cef537c40d14881d19b8bff71bd6baf5c94#diff-5de05088df1ea1a0745044571bfe9c684d033c18c156d81f2e0d4de50c836403L731-L732

Reply:
Yeah, that occurred to me as well.  I kept them in for completeness sake, but I suppose dead code is dead code and it's only serving to make the binary larger in the end. Fine, I'll kill it.